refinery/tool_manager/tests.py

```diff
@@ -1192,12 +1192,10 @@ def test_tool_container_removed_on_deletion(self):
         self.view(post_request)
 
         tool = Tool.objects.get(tool_definition__uuid=td.uuid)
+        tool_uuid = tool.uuid
```

Comment:
When I delete the Tool I lose access to its members, and I need its UUID for the DockerClientWrapper call below